packages/layout/src/footnotes/mapFootNotesToView.js

```diff
@@ -0,0 +1,59 @@
+
+import createInstance from '../node/createInstance';
+
+/**
+ *  maps footnote elements to a view node 
+ * 
+ * @param {*} footnotes 
+ * @returns a view node
+ */
+function mapFootNotesToView(footnotes) {
+  let processed = [
+    createInstance({
+      type: 'SVG',
+      props: {
+        style: {
+          height: 5,
+        },
+        children: [],
+      },
+    }),
+  ];
+
+  let line = createInstance({
+    type: 'LINE',
+    props: {
+      x1: 0,
+      x2: 1000,
+      y1: 0,
+      y2: 0,
+      strokeWidth: 1,
+      stroke: '#000000',
+    },
+  });
+
+  processed[0].children = [line];
```

Comment:
can we do this the other way?
i.e.: 
- Define `line`
- Define `processed` together with the children attribute set to include `line`